docs/cloud.md

```diff
@@ -0,0 +1,69 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in EKS and AKS.
+
+## Pre-requisites
+1. to run EKS cluster, install and configure aws cli, see
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html, and
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+2. to run AKS cluster, install and configure azure cli,
+   https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+3. install terraform
+   see https://learn.hashicorp.com/terraform/getting-started/install.html
+
+## Create an AKS cluster via terraform
+create or obtain a azure service principal. And set the following environment variables from the service principal. For instance in .bashrc
+```bash
+export TF_VAR_aks_client_id=YOUR_SERVICE_PRINCIPAL_ID
+export TF_VAR_aks_client_secret=YOUR_SERVICE_PRINCIPAL_ID
+export TF_VAR_aks_resource_group_name=YOUR_RESOURCE_GROUP_NAME
+```
+install Antrea cloud tools
+```bash
+make install-cloud-tool
+```
+create AKS cluster
+```bash
+~/terraform/aks
+```
+interact with AKS cluster
+```
+~/terraform/aks kubectl ... // issue kubectl commands to AKS cluster
+~/terraform/aks load ... // load local built images to AKS cluster
+~/terraform/aks destroy // destroy AKS cluster 
+```
+worker nodes are accessible via their external IP via ssh
+
+apply Antrea to AKS cluster
+```bash
+./hack/generate-manifest.sh --encap-mode networkPolicyOnly | ~/terraform/aks kubectl apply -f -
+```
+
+## Create an EKS cluster via terraform
+Ensures that you have permission to create EKS cluster, and have already
+created EKS cluster role as well as worker node profile.
+```bash
+export TF_VAR_eks_cluster_iam_role_name=YOUR_EKS_ROLE
+export TF_VAR_eks_iam_instance_profile_name=YOUR_EKS_WORKER_NODE_PROFILE
+export TF_VAR_eks_key_pair_name=YOUR_KEY_PAIR_TO_ACCESS_WORKER_NODE
+```
+install Antrea cloud tools
+```bash
+make install-cloud-tool
+```
+create EKS cluster
+```bash
+~/terraform/eks
```

Comment:
I would recommend using `~/terraform/eks create` for symmetry

Reply:
ok will do

Reply:
done